contracts/stake-cw20-reward-distributor/src/contract.rs

```diff
@@ -240,3 +244,34 @@ fn query_info(deps: Deps, env: Env) -> StdResult<InfoResponse> {
         balance: balance_info.balance,
     })
 }
+
+mod tests {
```

Comment:
Missing `#[cfg(test)]` above `mod tests`  (this is why there are a bunch of lint warnings about unused imports)

Reply:
It seems there's already a separate file for tests, so you can probably move the new test to `src/tests.rs`